lib/internal/Magento/Framework/Config/Dom.php

```diff
@@ -274,7 +274,8 @@ public static function validateDomDocument(
         $schema = self::$urnResolver->getRealPath($schema);
         libxml_use_internal_errors(true);
         try {
-            if (file_exists($schema)) {
+            // 4096 is the maximum length allowed by file_exists for the filename
+            if (strlen($schema) < 4096 && file_exists($schema)) {
```

Comment:
Yes ok. But actually it doesn't work. So… If I put myself at the same point of view as you… Why not merging this and removing the code after? You just said you will remove it. But fixing problems until then is still a good idea, isn't it?

Seems to be a good idea, isn't it?